independent-projects/bootstrap/app-model/src/main/java/io/quarkus/maven/dependency/ArtifactKey.java

```diff
@@ -34,4 +34,57 @@ default String toGacString() {
         }
         return buf.toString();
     }
+
+    static String[] split(String str, String[] parts, int fromIndex) {
```

Comment:
This method to me is an internal utility method, it doesn't feel like it belongs to the interface.

Reply:
Right, I moved the method to an `ArtifactKeyUtil` class in the `io.quarkus.util` package